src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelLockFileModule.java

```diff
@@ -178,10 +190,11 @@ public void afterCommand() {
       if (oldExtensionEntries != null) {
         // extension exists, add the new entry to the existing map
         extensionEntries =
-            new ImmutableMap.Builder<ModuleExtensionEvalFactors, LockFileModuleExtension>()
-                .putAll(oldExtensionEntries)
-                .put(factors, extension)
-                .buildKeepingLast();
+            ImmutableSortedMap.copyOf(
```

Comment:
nit - just use an ImmutableSortedMap.Builder?

Reply:
That builder class doesn't support `buildKeepingLast()`, so this seemed like the most direct workaround.

Reply:
I see!